### PR TITLE
Documentation edits

### PR DESCRIPTION
**Pull request by glasserc**
These are minor proofreading edits and changes to try to close out the leftover parts of #208: #563, #564, #565, #568, and #569.

### CODE REVIEW

File: docs/api/1.x/index.rst [hunk 1]
```diff
@@ -3,6 +3,30 @@
...
+   cliquet/backoff
+   cliquet/errors
+   cliquet/deprecation
+   buckets
+   collections
+   records
+   groups
+   permissions
+   synchronisation
+   cliquet/resource
```
Comment (leplatrem) [inline docs/api/1.x/index.rst]: Note: I wonder if we should not reorder them (ex. errors and deprecation after main objects buckets, collections...)

File: docs/api/1.x/permissions.rst [hunk 2]
```diff
@@ -193,12 +193,24 @@ Retrieve permissions
...
+==================
+
+An object's permissions can be modified at the same time as the object
+itself, using the same :ref:`PATCH <record-patch>` and :ref:`PUT
+<record-put>` methods discussed in :ref:`the Records section
+<records>`.
+
+.. note::
+
+   The user ID that updates *any* permissions is always added to the `write`
```
Comment (leplatrem) [inline docs/api/1.x/permissions.rst]: nitpick: for code use double backtick

File: docs/api/1.x/synchronisation.rst [hunk 3]
```diff
@@ -215,11 +215,13 @@ The **current implementation of reference** for offline-first records synchronis
...
-*ReadingList* project, such as `RL Web client`_ (React.js), `Android RL sync`_ (Java) or `Firefox RL client`_ (asm.js).
+*ReadingList* project. That project was abandoned, but you can still
+see the implementation of the `RL Web client`_ (React.js), `Android RL
+sync`_ (Java) or `Firefox RL client`_ (asm.js).
 
 .. _RL Web client: https://github.com/n1k0/readinglist-client/
-.. _Android RL Sync: https://hg.mozilla.org/releases/mozilla-beta/file/default/mobile/android/base/reading/
-.. _Firefox RL client: https://hg.mozilla.org/releases/mozilla-aurora/file/default/browser/components/readinglist
+.. _Android RL Sync: https://hg.mozilla.org/releases/mozilla-beta/file/FIREFOX_BETA_42_END/mobile/android/base/reading
+.. _Firefox RL client: https://hg.mozilla.org/releases/mozilla-aurora/file/FIREFOX_AURORA_41_END/browser/components/readinglist
```
Comment (leplatrem) [inline docs/api/1.x/synchronisation.rst]: Good catch :+1:

File: docs/contributing.rst [hunk 4]
```diff
@@ -29,27 +31,44 @@ Fix bugs
 --------
 
 Check out the `open bugs <https://github.com/Kinto/kinto/issues>`_ - anything
-tagged with the **[easy-pick]** label could be a good choice for newcomers.
+tagged with the |easy-pick label|_ could be a good choice for newcomers.
+
+.. |easy-pick label| replace:: **easy-pick** label
```
Comment (leplatrem) [inline docs/contributing.rst]: wao I had no idea we could do that for bold words within links :)  Awesome!

File: docs/overview.rst [hunk 5]
```diff
@@ -195,156 +195,5 @@ It can send a notification to clients using `WebSockets <https://en.wikipedia.or
 or fill a queue of messages in `Redis <http://redis.io/>`_ or execute any custom code of your choice,
 like for sending emails or pinging a third-party.
```
Comment (leplatrem) [inline docs/overview.rst]: question: should we add link to the FAQ ?

**Comment by leplatrem** [comment]
Awesome! thanks !

r+ : my concerns are mostly nitpicks that you can choose to ignore or address

**Comment by Natim** [comment]
Thanks :)

File: docs/api/1.x/permissions.rst (hunk 2 above)
Comment (glasserc) [inline docs/api/1.x/permissions.rst]: Good catch!

File: docs/contributing.rst (hunk 4 above)
Comment (glasserc) [inline docs/contributing.rst]: Like I wrote in https://github.com/Kinto/kinto/pull/574/commits/e46ba29e2875a6e3c59bdf06d453f12e46d338ff, it's kind of annoying, and I'm not sure it looks good in rendered HTML, but sure :)

File: docs/overview.rst (hunk 5 above)
Comment (glasserc) [inline docs/overview.rst]: Personally, I don't think so. The rendered form on Readthedocs has a link to the FAQ on the left side. I think most people look for/go to a FAQ when they have a specific question they want to see answered, which can happen at any point during the document (i.e. not just in the overview, but in concepts, or the tutorial).

File: docs/api/1.x/synchronisation.rst (hunk 3 above)
Comment (glasserc) [inline docs/api/1.x/synchronisation.rst]: I ran a link checker like @almet suggested in #564.

File: docs/api/1.x/index.rst (hunk 1 above)
Comment (glasserc) [inline docs/api/1.x/index.rst]: Done in https://github.com/Kinto/kinto/pull/574/commits/7737ff700a6a8764aae52c2d71fd2df7c4328854.

**Pull request merged**